app/subscriber/src/components/sidebar/advanced-search/interfaces/IAdvancedSearchFilter.ts

```diff
@@ -0,0 +1,12 @@
+export interface IAdvancedSearchFilter {
+  /** keeps track of which field the search term will query */
+  searchInField?: string;
```

Comment:
Normally using 'undefined' to manage state in a form will result in a lot of errors.  Should these default to empty string?